src/Microsoft.SqlTools.ServiceLayer/Localization/transXliff/sr.de.xlf

```diff
@@ -566,10 +566,11 @@
         <note/>
       </trans-unit>
       <trans-unit id="EditDataCreateScriptMissingValue">
-        <source>A required cell value is missing</source>
+        <source>The column '{0}' is defined as NOT NULL but was not given a value</source>
```

Comment:
In this case, we have added in a numerical parameter. We should test that this doesn't impact localized versions. @benrr101 can you help verify this? I'm just concerned if it throws an error. Otherwise it's OK to still have the old translation available.

Reply:
Running the same code as is defined in the sr.cs, it doesn't throw if a parameter is provided but not used in the formatting string. However, if the string expects parameters but none are provided, it will throw.